zipline/data/hdf5_daily_bars.py

```diff
@@ -726,21 +748,17 @@ def countries(self):
         return viewkeys(self._readers)
 
     def _country_code_for_assets(self, assets):
-        try:
-            country_codes = self._country_map.loc[assets]
-        except KeyError as exc:
-            raise_from(
-                NoDataForSid(
-                    'Assets not contained in daily pricing file: {}'.format(
-                        set(assets) - set(self._country_map)
-                    )
-                ),
-                exc
-            )
+        country_codes = self._country_map.get(assets)
```

Comment:
Why the switch from `.loc` to `.get` here? It looks like they have the same behavior:

```
In [6]: s
Out[6]:
a    1
b    2
dtype: int64

In [7]: s.get(['a', 'c'])
Out[7]:
a    1.0
c    NaN
dtype: float64

In [8]: s.loc[['a', 'c']]
Out[8]:
a    1.0
c    NaN
dtype: float64
```

Reply:
Oh, hmm, they differ when none of the keys are in the series...`loc` raises a KeyError, but `get` returns an all-nan series.

Reply:
And in newer pandas, `get` returns None!

Reply:
🤔